README.md

```diff
@@ -58,6 +59,18 @@ To run the application on an android emulator:
 
 **[Back to top](#table-of-contents)**
 
+### Debugging
+
+In order to debug the app, `remote debugging` must be enabled on the device or simulator.
+
+On the iPhone simulator:
+1. Press `Cmd + D` to open the development menu
+1. Click `Debug Remote JS`
+
+If remote debugging is unavailable, ensure that `production mode` is not enabled (for instance, in the Metro bundler)
```

Comment:
Mention that the browser's metro bundler has a toggle button called `production mode`